devo/sender/lookup.py

```diff
@@ -353,8 +348,8 @@ def list_to_headers(headers=None, key=None,
                                                         else type_of_key)
         elif key is not None:
             out = '[{"%s":{"type":"%s","key":true}}' % (key, types[key_index]
-                                                        if key_index and types
-                                                        else type_of_key)
+            if key_index and types
```

Comment:
I think this should be indented to the `types[key_index]` field to avoid confusion.